CalibTracker/SiPixelLorentzAngle/src/SiPixelLorentzAnglePCLHarvester.cc

```diff
@@ -102,6 +102,7 @@ class SiPixelLorentzAnglePCLHarvester : public DQMEDHarvester {
   const std::string recordName_;
   std::unique_ptr<TF1> f1;
   float width_;
+  float theMagField{0.f};
```

Comment:
please use trailing underscore for data member as per CMS code rules.

Reply:
Done